Code-Hills/codehills-be
# Chore - Set up a node.js project

#### What does this PR do?
> - This PR sets up a node.js project

#### How should this be manually tested?
> - clone this repository
> - checkout to this branch `ch-project-setup`
> - install the dependencies `npm install`
> - Start the dev server `npm run dev`
> - Go to `http://localhost:3000` and you should see `Welcome to codehills backend!`
